lib/overlay.js

```diff
@@ -97,34 +113,51 @@ export const Overlay = ({ open, onClose, children, contentProps, ...props }) =>
   return (
     <OverlayWrap data-module="Overlay" onClick={onClickBackground} {...props}>
       <FixedBody />
-      <OverlayContent {...contentProps}>{children({ onClose, first, last, focusedOnMount })}</OverlayContent>
+      <OverlayContent variant={variant} {...contentProps}>
+        {children({ onClose, first, last, focusedOnMount })}
+      </OverlayContent>
     </OverlayWrap>
   );
 };
+
 Overlay.propTypes = {
   open: PropTypes.bool.isRequired,
   children: PropTypes.func.isRequired,
   onClose: PropTypes.func.isRequired,
   contentProps: PropTypes.object,
+  variant: PropTypes.oneOf(Object.keys(variants)),
 };
 
+const ButtonContainer = styled.div`
+  margin: var(--space-1) auto;
+  & > * {
+    margin: 0 var(--space-1) var(--space-1) 0;
+  }
+`;
+
 export const useOverlay = () => {
-  const [open, setOpen] = React.useState(false);
+  const [openNormal, setOpenNormal] = React.useState(false);
+  const [openBanner, setOpenBanner] = React.useState(false);
   const toggleRef = React.useRef();
-  const onOpen = () => setOpen(true);
+  const onOpenNormal = () => setOpenNormal(true);
+  const onOpenBanner = () => setOpenBanner(true);
   const onClose = () => {
-    setOpen(false);
+    setOpenNormal(false);
+    setOpenBanner(false);
     toggleRef.current.focus();
   };
-  return { open, onOpen, onClose, toggleRef };
+  return { openNormal, onOpenNormal, openBanner, onOpenBanner, onClose, toggleRef };
```

Comment:
`useOverlay` is actually exported for use outside shared-components, so we shouldn't add a second set of state values since it's meant to handle a single overlay. We can call it twice in `StoryOverlay`, though, once for each overlay.